src/Readme.md

```diff
@@ -0,0 +1,12 @@
+<!-- Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. -->
+
+# src
+=========
+
+src folder content is very similar to that of Tryouts folder, but it is more final and more perfect. 
```

Comment:
Since 'src' is the root folder of the actual source code for the platform I wouldn't start the description by saying 'this is similar to something else, you should check out that first'. For people interested in the source code this would be the landing page.

Reply:
good point

Reply:
removed the line